accelerometer/circadianRhythms.py

```diff
@@ -0,0 +1,125 @@
+"""Module to support calculation of metrics of circadian rhythm from acc data"""
+
+import gzip
+import numpy as np
+import pandas as pd
+import pytz
+import sys
+import scipy as sp
+from scipy import fftpack
+from datetime import timedelta
+
```

Comment:
Minor pedantic request - can you add in 3 spaces between functions/methods in this file?

Reply:
fixed in ce9842906efbd024c24d668b2d3bd0b3f429f559